consul-lambda-registrator/event_test.go

```diff
@@ -90,22 +90,20 @@ func TestUpsertAndDelete(t *testing.T) {
 }
 
 func assertConsulState(t *testing.T, consulClient *api.Client, env Environment, event UpsertEvent, count int) {
-	var queryOptions *api.QueryOptions
+	queryOptions := &api.QueryOptions{Datacenter: event.Datacenter}
```

Comment:
I wonder if it makes sense to make `event.QueryOptions` and/or `event.WriteOptions` to get rid of some of this boilerplate code. This is fine for now though.